src/sage/doctest/forker.py

```diff
@@ -1467,7 +1475,7 @@ def report_failure(self, out, test, example, got, globs):
                     self._fakeout.start_spoofing()
             return returnval
 
-    def report_overtime(self, out, test, example, got, *, check_duration=0):
+    def report_overtime(self, out, test, example, got, *, check_timer=None):
```

Comment:
The old keyword should be deprecated using `sage.misc.decorators.rename_keyword`, rather than just changed.

Reply:
The keyword wasn't simply renamed -- the type of argument that it takes was changed too. Typically that would be an unfriendly thing to do, but this is essentially a private method. No other part of the sage library uses it (it's referenced exactly once, by `self`), and since it's part of sage's custom doctest runner, it's hard to imagine a third party relying on these internal details. Frankly I'd like to change it to an underscore method now that you mention it.

Reply:
> The keyword wasn't simply renamed -- the type of argument that it takes was changed too. Typically that would be an unfriendly thing to do, but this is essentially a private method. No other part of the sage library uses it (it's referenced exactly once, by `self`), and since it's part of sage's custom doctest runner, it's hard to imagine a third party relying on these internal details. 

Cool. Sage policy is still to deprecate it. So deprecate the old argument and add a new one.

Reply:
There's no way to translate the old argument into the new one. If we keep the old name but ignore its argument, then any code using this function will still be immediately broken, only more quietly and with more steps.

Reply:
If you feel that strongly about not changing this method's signature, I can copy & paste the entire function into a new private method `_report_overtime`. And then we should do the same for `_report_success`. That will leave the old methods intact to be deprecated, but I really think we would be wasting our time.

It's going to take time to put the old functions back and to add the deprecation warnings. It will take time to re-review those changes. It's going to take time to run the doctests for two unused functions in the interim. And then eventually someone has to spend the time to remove the functions, and someone else has to spend the time to review that.

Unless there's an (even remotely) plausible reason why anyone would be calling this function in their own code, it's justifiable to bend the deprecation rule. Its intent has been respected. And to do otherwise is wasting our most valuable resource.